Sources/swift-doctest/main.swift

```diff
@@ -2,6 +2,15 @@ import ArgumentParser
 import DocTest
 import Foundation
 import TAP
+// pattern borrowed upstream from swift:
+// https://github.com/apple/swift/blob/master/test/Interpreter/SDK/libc.swift#L10-L14
+#if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
+  import Darwin
+#elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android) || os(Cygwin) || os(Haiku)
+  import Glibc
+#elseif os(Windows)
+  import MSVCRT
+#endif
```

Comment:
```suggestion
// Pattern borrowed upstream from Swift: 
// https://github.com/apple/swift/blob/87d3b4d984281b113ffad503cdb1d82b9f0ae5b9/test/Interpreter/SDK/libc.swift#L12-L17
#if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
  import Darwin
#elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android) || os(Cygwin) || os(Haiku)
  import Glibc
#elseif os(Windows)
  import MSVCRT
#endif
```

Reply:
Using a permalink ensures that file and line references are stable (when I visited the provided URL, the highlighted portion was off by a couple lines)